src/__test__/CountriesReducer.test.js

```diff
@@ -0,0 +1,30 @@
+import CountriesReducer from '../Redux/CountriesReducer';
```

Comment:
- [ ] Great job so far 👏🏻 However I recommended you add more unit tests for several components of the app as requested on the project requiements. ✍🏻